onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/handler/ModuleHandler.java

```diff
@@ -317,4 +317,8 @@ private Set<String> gatherModulesIris(List<FiboModule> modules) {
     }
     return iris;
   }
+
+    void clearModules() {
```

Comment:
Instead of this, I would add a method name `updateModules` that would override modules.

Reply:
When i renamed method i moved some code to more appropriate method